.github/workflows/helm-release.yaml

```diff
@@ -2,17 +2,22 @@
 name: Release Prefect Operator Helm Chart
 
 "on":
-  workflow_dispatch: {}
-  workflow_call: {}
+  workflow_call:
```

Comment:
no need for workflow_dispatch anymore -- since we will be building dev charts on every merge to main